liberty-maven-plugin/src/it/binary-scanner-it/run-binary-scanner-it

```diff
@@ -10,14 +11,22 @@
 # five minutes for each test.
 cd liberty-maven-plugin/
 cp src/it/binary-scanner-it/pom.xml /tmp
-for i in 3.7 3.7.1
+restorePOM () {
+    cp /tmp/pom.xml src/it/binary-scanner-it/pom.xml
+}
+trap restorePOM SIGINT
+for i in 3.9
 do
-sed "s/@pom.version@/$i/g" </tmp/pom.xml >src/it/binary-scanner-it/pom.xml
-mvn install verify -Ponline-its -Dinvoker.streamLogs=true -Druntime=ol -DruntimeVersion=23.0.0.1 -Dinvoker.test=binary-scanner-it
 echo !!
+echo !! About to test using Liberty Maven Plugin version $i.
+echo !! Each test runs for 5-10 minutes and then pauses here
+echo !! so you can check the results.
+echo !! You should already have followed the instructions in
+echo !! the comments at the top of this script. 
 echo !! PRESS ENTER TO CONTINUE
 echo !!
-# Allow time to inspect the pass/fail status before continuing.
 read j
+sed "s/@pom.version@/$i/g" </tmp/pom.xml >src/it/binary-scanner-it/pom.xml
+mvn install verify -Ponline-its -Dinvoker.streamLogs=true -Druntime=ol -DruntimeVersion=23.0.0.1 -Dinvoker.test=binary-scanner-it
```

Comment:
is there a reason we are using OL runtime version 23.0.0.1? Should the runtime version be a constant set at the top of the script?

Reply:
No, there is no reason. Last time I updated from a 2022 Liberty to 2023 and just left it this time. I'll make it a variable.